packages/agent/src/actor.ts

```diff
@@ -364,7 +376,7 @@ function _createActorMethod(
         effectiveCanisterId: ecid,
       });
 
-      if (!response.ok) {
+      if (!response.ok || response.body /* IC-1462 */) {
```

Comment:
Would you please describe a little more here why we would treat "the response has a body" as "Update call rejected" ?  I'm reading the description for IC-1462 and don't understand.

Reply:
From the [spec](https://github.com/dfinity/interface-spec/blob/master/spec/index.md#request-call-http-call):

<img width="1041" alt="image" src="https://github.com/dfinity/agent-js/assets/522097/1d009bdc-16ed-4a8a-a032-2fdf6c3c59ce">

I don't fully understand the reasoning for this change, but this is indeed the correct behavior for the latest `dfx` release.